.stylelintrc

```diff
@@ -0,0 +1,28 @@
+{
+    "extends": "stylelint-config-recommended-scss",
```

Comment:
why not use the airbnb standard for sass linting too? https://github.com/airbnb/css